src/components/document.js

```diff
@@ -177,6 +178,12 @@ function Document({
       children
         ? jsx("main", null, children)
         : jsx("main", { dangerouslySetInnerHTML: { __html: content } }),
+      config.analytics &&
+        jsx("script", {
+          async: true,
```

Comment:
Are you sure this `async` attribute could not cause `load-analytics` to throw because it loaded before gtag?

Reply:
![imagen](https://user-images.githubusercontent.com/13140456/56105790-16e71500-5f0c-11e9-9cf8-bc85b49b9b9e.png)

```suggestion
          async: true,
```

I do not think so, it's the script with the same structure that Google analytics provides.

Reply:
I do not think so, it's the script with the same structure that Google analytics provides.